frontend/src/container/MetricsTable/Columns/MetricsColumn.ts

```diff
@@ -0,0 +1,48 @@
+import type { ColumnsType } from 'antd/es/table';
+import { ServicesList } from 'types/api/metrics/getService';
+
+import {
+	ColumnDataIndex,
+	ColumnKey,
+	ColumnTitle,
+	ColumnWidth,
+	SORTING_ORDER,
+} from './ColumnContants';
+import { getColumnSearchProps } from './GetColumnSearchProps';
+
+export const columns = (search: string): ColumnsType<DataProps> => [
+	{
+		title: ColumnTitle.Application,
+		dataIndex: ColumnDataIndex.Application,
+		width: ColumnWidth.Application,
+		key: ColumnKey.Application,
+		...getColumnSearchProps('serviceName', search),
```

Comment:
this is not scalable as we have to make a function which dynamically add the props to the column of the table so that we can use everywhere